### PR TITLE
Drop references to the committee

### PR DESCRIPTION
**Pull request by gelisam**
This is not a proposed change to the ecosystem, but a proposed change to this repository. As such, perhaps-controversially, this PR changes the text of an already-accepted proposal, the proposal process proposal. Please let me know if you would prefer if I instead wrote a brand new proposal process proposal, or if I instead wrote a proposal for changing the proposal process proposal.

-----

[There is no ecosystem committee](https://github.com/haskell/ecosystem-proposals/pull/3#issuecomment-446285305). The fact that the proposal process proposal mentions a committee is thus misleading, giving the impression that it was possible to make a change happen in the ecosystem by making a pull request in this repository and waiting for "the ecosystem committee" to approve it. Since there is no committee, this leads to frustration as pull requests are never accepted.

I edited those parts of the readme and the proposal process proposal to emphasize the importance of getting buy-in from the representatives of the projects which will implement the change. There was already a paragraph about this requirement, so this is nothing new, I merely made it more prominent in the instructions.

The only new bit this PR introduces is to specify how a pull request gets merged instead: if it's not when the committee accepts it, then when? In the end, it doesn't really matter whether a proposal is marked as accepted or not; if the representatives of the involved projects all agree to make the change happen, it will happen even if the proposal is marked as rejected, and if there is no buy-in from the relevant projects, the change will not happen even if the proposal is marked as accepted. That green seal of approval is thus only cosmetic, and so it doesn't really matter what the criteria is. For simplicity, I just wrote that the author should write a comment saying that they now consider the proposal to be accepted, and that someone with the commit bit will then merge the PR. I don't really care about that particular change, I just feel like the proposal should say _something_ about how the PRs get accepted, otherwise they'll stay open forever.

### CODE REVIEW

File: proposals/0000-proposal-process.rst [hunk 1]
```diff
@@ -50,87 +54,40 @@ Each proposal goes through the following stages:
...
+   it is important to get buy-in first.)
 
-8. During implementation, it is very likely that the specification will be refined.
+7. During implementation, it is very likely that the specification will be refined.
    The implementor should keep the specification up to date over the course of
    the implementation.
 
-9. If a proposal affects multiple components of the ecosystem and requires
-   coordination between them, there must be explicit buy-in from all affected
-   parties that they agree with the proposal.
```
Comment (gelisam) [inline proposals/0000-proposal-process.rst:89]: I deleted this paragraph because its contents is now emphasized elsewhere many times.

**Comment by gbaz** [comment]
Thanks. I think this helps clarify things a bit. Since there's no one formal group to approve this, my impulse is to just let it sit a bit, and see what discussion emerges. If it seems appropriate after that discussion (or after a lack of discussion), then I'll merge.

**Comment by mitchellwrosen** [comment]
Without a committee to decide whether we should accept this PR that points out there is exists no committee that can accept PRs, I'm afraid the first step ought to be to form a committee with the authority to do so, at which point this PR should be rejected for being inaccurate.

**Comment by gbaz** [comment]
@mitchellwrosen I can't tell if you're joking, but if you aren't, that's inaccurate. The description of the repo was never correct to begin with. There's no reason we need committees for everything, since this is all part of a protracted and confused bootstrap process. (There's also no basis for the correct way to form a committee in this case, since, again, the bootstrap process was incomplete in setting this whole thing up). If you are joking, sorry for being dense about this, but enough people have been confused by all this that I think it's important to be very explicit and clear.

**Comment by mitchellwrosen** [comment]
Yes it was totally a joke, sorry that was not clear. Needed a smiley face or something :)

**Comment by alanz** [comment]
FWIW, when I put this thing together I basically copied over the process from the ghc-proposals repo. And at the time there was a *huge* amount of unhappiness in the community in general, so I did not press for this to be finalised, as it would have been just another point of conflict.

But my original reason was to provide a single well-known place where contentious things could be discussed, as prior to that there would be discussion popping up on email, github issues, reddit, etc, and the same arguments being had each time.

So it being a venue for discussion only, without any particular authority to enforce a decision is exactly what it should be. In my opinion.

**Comment by gelisam** [comment]
> Since there's no one formal group to approve this, my impulse is to just let it sit a bit, and see what discussion emerges. If it seems appropriate after that discussion (or after a lack of discussion), then I'll merge.

Has it been long enough yet to consider merging?

**Comment by alanz** [comment]
LGTM.  I suggest we wait say 24 hours to see if anything pops up, then merge.

**Comment by gelisam** [comment]
All right, it's been 24 hours and you've given me the commit bit, so I'll merge!

**Pull request merged**